tests/test_formats.py

```diff
@@ -1115,6 +1115,51 @@ def test_format_number_with_sep_dec_marks():
     assert x == ["12.345.678,12346", "1,00000", "0,00000", "\u2212" + "12.345.678,12346"]
 
 
+# ------------------------------------------------------------------------------
+# Tests of `fmt_scientific()`
+# ------------------------------------------------------------------------------
+
+FMT_SCIENTIFIC_CASES: list[tuple[dict[str, Any], list[str]]] = [
```

Comment:
Do you mind making the cases each only a single input, and a single output? That way, readers can tell which outputs of each test case matter (right now it's clear the inputs cross with kwargs, so its not clear if each output is important, or there as a side effect of crossing).

(This would just require a x_in parameter in the test signature)